internal/local/app.go

```diff
@@ -221,10 +225,11 @@ func (a *App) LoadConfig() error {
 		a.AppData = &AppConfigJSON{}
 	case FileStitch:
 		a.AppData = &AppStitchJSON{}
+	default:
+		return errInvalidConfigFile(a.Config.String())
```

Comment:
considering the invalid config file error is only used again in a test, I think we can avoid the helper/abstraction for it and simplify things a bit in the process:

```go
default:
  return fmt.Errorf("invalid config file: %s", a.Config)
```

and then simply in the test assert things equal `errors.New("invalid config file: bogus.json")`